plugin-server/functional_tests/ingestion.test.ts

```diff
@@ -317,108 +272,197 @@ test.concurrent('consumer updates timestamp exported to prometheus', async () =>
-            )
-        }, 10000)
-
-        // We then have the user identify themselves, but on e.g. a different
-        // device and hence a different anonymous id.
-        const initialIdentifyEventId = new UUIDT().toString()
-        await capture(producer, teamId, personIdentifier, initialIdentifyEventId, '$identify', {
-            $anon_distinct_id: secondDistinctId,
-            distinct_id: personIdentifier,
-        })
-        await waitForExpect(async () => {
-            const persons = await fetchPersons(clickHouseClient, teamId)
-            expect(persons).toContainEqual(
+                }),
                 expect.objectContaining({
-                    properties: expect.objectContaining({ $creator_event_uuid: initialIdentifyEventId }),
-                })
-            )
-        }, 10000)
-
-        // Then we create another event with the initial anonymous distinct id,
-        // shortly followed by another identify event but this time with the
-        // initial anonymous distinct id
-        const uuidOfEventThatShouldBeIdentified = new UUIDT().toString()
-        await capture(producer, teamId, initialDistinctId, uuidOfEventThatShouldBeIdentified, 'custom event')
-
-        const uuidOfIdentifyEvent = new UUIDT().toString()
-        await capture(producer, teamId, personIdentifier, uuidOfIdentifyEvent, '$identify', {
-            distinct_id: personIdentifier,
-            $anon_distinct_id: initialDistinctId,
-        })
+                    properties: expect.objectContaining({ $creator_event_uuid: secondEventId }),
+                }),
+            ])
+        )
+    }, 10000)
+
+    // Then we identify both ids
+    const uuidOfFirstIdentifyEvent = new UUIDT().toString()
+    await capture(producer, teamId, personIdentifier, uuidOfFirstIdentifyEvent, '$identify', {
+        distinct_id: personIdentifier,
+        $anon_distinct_id: initialDistinctId,
+    })
+    const uuidOfSecondIdentifyEvent = new UUIDT().toString()
+    await capture(producer, teamId, personIdentifier, uuidOfSecondIdentifyEvent, '$identify', {
+        distinct_id: personIdentifier,
+        $anon_distinct_id: secondEventId,
+    })
+
+    let events: ClickHouseEvent[] = []
+    await waitForExpect(async () => {
+        events = await fetchEvents(clickHouseClient, teamId)
+        expect(events.length).toBe(4)
+    }, 10000)
+
+    events.forEach((event) => {
+        expect(event?.person_id).toBeDefined()
+    })
 
-        await waitForExpect(async () => {
-            const [anonymousEvent] = await fetchEvents(clickHouseClient, teamId, uuidOfEventThatShouldBeIdentified)
-            const [identifyEvent] = await fetchEvents(clickHouseClient, teamId, uuidOfIdentifyEvent)
-            expect(anonymousEvent?.person_id).toBeDefined()
-            expect(identifyEvent?.person_id).toBeDefined()
-            expect(anonymousEvent.person_id).toEqual(identifyEvent.person_id)
-        }, 10000)
+    if (testIfPoEEmbraceJoinEnabled) {
```

Comment:
Can we keep this as a skipped test rather than registering as successful? Not a fan of tests that have conditionals, it's potentially easy to think you've tested something when you haven't.